tools/cluster_mgr.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import math
+import redis
+import subprocess
+import time
+
+'''
+To install: pip install -r requirements.txt
+'''
+
+
+class Node:
+    def __init__(self, port, admin_port):
+        self.port = port
+        self.admin_port = admin_port
+
+    id = ''
+
+
+def start_node(node, threads):
+    f = open(f'/tmp/dfly.cluster.node.{node.port}.log', 'w')
+    print(f'- Log file for node {node.port}: {f.name}')
+    subprocess.Popen(['../build-dbg/dragonfly', f'--port={node.port}',
+                      f'--admin_port={node.admin_port}', '--cluster_mode=yes', f'--proactor_threads={threads}',
+                      '--dbfilename=', f'--logtostderr'], stderr=f)
+
+
+def send_command(node, command):
+    for i in range(0, 5):
+        try:
+            client = redis.Redis(decode_responses=True,
+                                 host="localhost", port=node.admin_port)
+            result = client.execute_command(*command)
+            client.close()
+            return result
+        except:
+            time.sleep(0.1)
+
```

Comment:
I think you can just create a single client and call ping

Reply:
But on every call I, potentially, connect to a different node

Reply:
How? The function accepts a single node, or is it mutated somehow?

Reply:
Oh, sorry, now I understand what you meant.